tasks/main.yml

```diff
@@ -12,6 +12,9 @@
 - import_tasks: ssh.yml
   tags: [openwisp2, ssh]
 
+- import_tasks: system.yml
```

Comment:
Can you fix the linter warnings that appear after adding this?
Later on we can fix all the other remaining issues too (I will open an issue).